examples/multi-project/README.md

```diff
@@ -115,11 +117,14 @@ jobs:
           infracost output --path="infracost_jsons/*.json" --format=json --out-file=/tmp/infracost_combined.json
         displayName: Combine the results
 
-      - task: InfracostComment@0
-        displayName: Post the comment
-        inputs:
-          githubToken: $(githubToken)
-          path: /tmp/infracost_combined.json
-          behavior: update # Create a single comment and update it. See https://github.com/infracost/infracost-azure-devops#infracostcomment for other options
+      - bash: |
+          # Create a single comment and update it. See https://www.infracost.io/docs/features/cli_commands/#comment-on-pull-requests for other options
+          infracost comment github \
+          --path /tmp/infracost_combined.json \
```

Comment:
```suggestion
          --path="infracost_jsons/*.json" \
```

Reply:
done in https://github.com/infracost/infracost-azure-devops/pull/45/commits/51e7994d795d728acc2fcd1bef0007f8f0344df2